edx_sga/tests.py

```diff
@@ -28,22 +33,33 @@ def __eq__(self, other):
 
 
 class DummyUpload(object):
-
+    """
+    Upload and read file.
+    """
     def __init__(self, path, name):
         self.stream = open(path, 'rb')
         self.name = name
         self.size = os.path.getsize(path)
 
-    def read(self, n=None):
-        return self.stream.read(n)
+    def read(self, number_of_bytes=None):
+        """
+        Read data from file.
+        """
+        return self.stream.read(number_of_bytes)
 
-    def seek(self, n):
-        return self.stream.seek(n)
+    def seek(self, offset):
+        """
+        Create a SGA block with mock data.
+        """
+        return self.stream.seek(offset)
 
 
 class StaffGradedAssignmentXblockTests(unittest.TestCase):
-
+    """
+    Create a SGA block with mock data.
+    """
     def setUp(self):
+        super(StaffGradedAssignmentXblockTests, self).setUp()
```

Comment:
Add description of what the setUp does, i.e. "Creates a test course ID, mocks the runtime, and creates a fake storage engine for use in all tests"